docs/resources/mws_ncc_private_endpoint_rule.md

```diff
@@ -0,0 +1,71 @@
+---
+subcategory: "Deployment"
+---
+# databricks_mws_ncc_private_endpoint_rule Resource
+
+-> **Note** Initialize provider with `alias = "account"`, `host = "https://accounts.azuredatabricks.net"` and use `provider = databricks.account` for all `databricks_mws_*` resources.
+
+-> **Public Preview** This feature is in [Public Preview](https://docs.databricks.com/release-notes/release-types.html) in Azure.
+
+Allows you to create a [Network Connectivity Config] that can be used as part of a [databricks_mws_workspaces](mws_workspaces.md) resource to create a [Databricks Workspace that leverages serverless network connectivity configs](https://learn.microsoft.com/en-us/azure/databricks/sql/admin/serverless-firewall).
```

Comment:
Not to create NCC, but add a private endpoint to it?

Reply:
done